src/components/Info/Index.js

```diff
@@ -1,19 +1,60 @@
+import {
+  InfoContainerCustom,
+  TextContainer,
+  TextInfoBox,
+  DescriptionText,
+  TextLabel,
+  AbilitiesContainer,
+  InfoRequestCustom,
+  HandlerContainer,
+} from "./styled";
+
 export default function Info({
   pokemonInfo,
   setPokemonInfo,
   pokemonDescription,
   setPokemonDescription,
 }) {
   return (
-    <div>
-      <p>{pokemonInfo.weight}</p>
-      <p>{pokemonInfo.height}</p>
-      <p>{pokemonInfo.abilities && pokemonInfo.abilities[0].ability.name}</p>
-      <p>{pokemonInfo.abilities && pokemonInfo.abilities[1].ability.name}</p>
-      <p>
-        {pokemonDescription.flavor_text_entries &&
-          pokemonDescription.flavor_text_entries[0].flavor_text}
-      </p>
-    </div>
+    <HandlerContainer>
+      <InfoContainerCustom>
+        <TextContainer>
+          <TextInfoBox>
+            <TextLabel>Weight</TextLabel>
+            <InfoRequestCustom>{pokemonInfo.weight} kg</InfoRequestCustom>
+          </TextInfoBox>
+          <TextInfoBox>
+            <TextLabel>Height</TextLabel>
+            <InfoRequestCustom>{pokemonInfo.height} m</InfoRequestCustom>
+          </TextInfoBox>
+          <TextInfoBox>
+            <TextLabel>Abilities</TextLabel>
+            <AbilitiesContainer>
+              {pokemonInfo.abilities.map((ability, index) => {
+                return <p>{ability.ability.name}</p>;
+              })}
+            </AbilitiesContainer>
+          </TextInfoBox>
+          <TextInfoBox>
+            <DescriptionText>
+              <TextLabel>Description</TextLabel>
+              {pokemonDescription.flavor_text_entries.map(
+                (description, index) => {
+                  return (
+                    <p>
+                      {description.flavor_text === [0]
```

Comment:
Nice very good practice !
What you tried here was to test if description.flavor_text exist (description.flavor_text !== null or just with description.flavor_text)
You just did pretty weird but keep continue !